src/components/App.tsx

```diff
@@ -3042,7 +3043,13 @@ class App extends React.Component<AppProps, AppState> {
           scenePointerX,
           scenePointerY,
         );
-        if (hoverPointIndex >= 0) {
+        midPointHovered = !!LinearElementEditor.isHittingMidPoint(
```

Comment:
```suggestion
        midPointHovered = LinearElementEditor.isHittingMidPoint(
```

let's make this always return a `boolean` instead